src/helpers/formulaHelper.js

```diff
@@ -0,0 +1,187 @@
+const formulas = require('./formulas.json')
+
+const TOKEN_TYPE = {
+    Number: Symbol("number"),
+    Variable: Symbol("variable"),
+    Function: Symbol("function"),
+    Operator: Symbol("operator")
+};
+
+const FUNCTION_TOKENS = [
+    "MIN",
+    "MAX",
+    "ROUND"
+];
+
+function getFormulaByKey(key) {
+    return formulas[key];
+}
+
+function parse(formulaObj) {
+    const constants = formulaObj.constants;
+    let tokens = formulaObj.formula.split('');
+
+    let resultStack = [];
+    let parseStack = [];
+
+    for (let i = 0; i < tokens.length; i++) {
```

Comment:
Can we extract this for loop into a function?

Reply:
I'll see what I can do